static/js/directives/ui/quota-management-view.js

```diff
@@ -312,6 +314,16 @@ angular.module('quay').directive('quotaManagementView', function () {
           return true;
         }
 
+        if ($scope.newLimitConfig['limit_percent'] != null && isNaN(parseInt($scope.newLimitConfig['limit_percent']))) {
```

Comment:
parseInt here may not work if the value is decimal. check for decimals first then parse

Reply:
This is not a check for decimals. This is to check if the input is an integer.

Reply:
yes but this is a if statement what happens if parseInt is given a decimal. In other languages it would fail and throw an exception here your if statement may not evaluate.

Reply:
So, In javascript, [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) converts a string type to integer. parseInt(-12.23) gives -12. I am using this to catch exponential values like `"e12.23"` which should return an error. parseInt("e12.23") returns Nan (Not a number) so using `isNan` to raise an error here.